Content.Tests/DMTests.cs

```diff
@@ -169,8 +169,7 @@ private static IEnumerable<object[]> GetTests()
             DMTestFlags testFlags = DMTestFlags.NoError;
 
             using (StreamReader reader = new StreamReader(sourceFile)) {
-                string firstLine = reader.ReadLine();
-
+                string firstLine = reader.ReadLine() ?? throw new System.Diagnostics.UnreachableException("reader.ReadLine() returned null");
```

Comment:
This is not necessarily unreachable, is it? Could be an empty file.

Reply:
Throw a error pointing to the file when its null then? I kinda want the compiler to not null warn for this

Reply:
Could return early if the file is empty
```cs
string? firstLine = reader.ReadLine();
if (firstLine == null)
    return testFlags;
```

Reply:
An error works too though